GVFS/GVFS.Common/Paths.Shared.cs

```diff
@@ -68,5 +68,55 @@ public static string ConvertPathToGitFormat(string path)
         {
             return path.Replace(Path.DirectorySeparatorChar, GVFSConstants.GitPathSeparator);
         }
+
+        public static string GetRelativePath(string relativeTo, string path)
+        {
+            if (!Path.IsPathRooted(relativeTo))
+            {
+                throw new ArgumentException("Path must be absolute.", nameof(relativeTo));
+            }
+
+            if (!Path.IsPathRooted(path))
+            {
+                throw new ArgumentException("Path must be absolute.", nameof(path));
+            }
+
+            // Normalize paths

```

Comment:
What do you mean by "normalize"? You already checked that they are full paths up above, so isn't `GetFullPath` going to be a noop? Also we have a method `IPlatformFileSystem.TryGetNormalizedPath`, which on Windows ends up calling `GetFinalPathNameByHandle` to normalize the path. 

I'm not really sure I understand the goal of this `GetRelativePath` method though. In general, paths are much more complicated than they appear and converting them in this way seems likely to be very error-prone.